Dockerfile.aarch64

```diff
@@ -23,7 +23,7 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM aarch64/ubuntu:trusty
+FROM aarch64/ubuntu:vivid
```

Comment:
You can update this PR, by doing `git commit -amend` and doing a force-push (`git push -f ....`). That will update this PR (so no need to open a new PR)